src/Universum/Nub.hs

```diff
@@ -32,10 +32,26 @@ import Data.Eq (Eq)
 import Data.Hashable (Hashable)
 import Data.HashSet as HashSet
 import Data.Ord (Ord)
-import Prelude ((.))
+import Data.Set (Set)
+import Prelude (Bool, Char, (.))
```

Comment:
It looks like this will print some warnings... Maybe yuu could write `Prelude.Bool` in type signature if possible? Also, its better to imports things from `universum` modules if possible to test package by itself...

Reply:
The thing is it's not used in this annotations straight but it still needs this imports and I don't know how to deal with it.

Reply:
Will it work if you use `Prelude.Bool` instead of just `Bool` and won't import?

If doesn't you can just disable redundant imports warning in this module only I guess...

Reply:
I don't know where to put `Prelude.Bool` if there is no `Bool` in the code itself.. :disappointed: 

Probably we should keep warnings to not forget to find the solution for this later?

Reply:
Here what I saw: https://github.com/ucsd-progsys/liquidhaskell/issues/1019